website/docs/guides/shebang.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Shebang
+sidebar_position: 30
+---
+
+This command is equivalent to `run`, but it changes the way Scala CLI parses arguments in order to be
+compatible with shebang scripts.
+
+
+### In shebang script
+
+Before proceeding, let's discuss how `scala-cli` works in a script without the `shebang` command.
+Here is a simple hello script with a shebang header:
+
+```scala title=hello.sc
+#!/usr/bin/env scala-cli -S 3
+
+println(args.size)
+println(args.headOption)
+```
+
+And it almost works correctly.
+
+```bash
+./hello.sc    
+```
+
+```text
+0
+None
+```
+
+And it also works.
+
+```bash
+./hello.sc -- Hello World
+2
+Some(Hello)
+```
+
+But note that the extra `--` must be added to make it work. If it is not supplied, the result is:
+
+```bash
+./hello.sc Hello World
+Hello: not found
+World: not found
+```
+
+If we modify our script slightly and use the `shebang` command, we will get this effect.
+
+```scala title=hello.sc
+#!/usr/bin/env scala-cli shebang -S 3
+
+println(args.size)
+println(args.headOption)
+```
+
+```bash
+./hello.sc Hello World
+2
+Some(Hello)
+```
+
+### In command line
+
+Let's now see how the shebang command works in the command line.
+
+```scala title=Main.scala 
+object Main {
+  def main(args: Array[String]): Unit = println(args.mkString)
+}  
+```
+```bash                                                                                                                                                                                                                                                                
+scala-cli shebang Main.scala Hello world
+# Hello world
+```
+
+Please note that `shebang` change in how arguments are parsed means that every options after the first input will be treated as
+an argument to the app.
+
+If we try to do the same with the run command, we get the following error:
+```bash ignore
+scala-cli run Main.scala Hello world
+# [error]  Hello: not found
+#  : not found
+# world: not found
+```
```

Comment:
````suggestion
<ChainedSnippets>

```bash ignore
scala-cli run Main.scala Hello world
```

```text
[error]  Hello: not found
 : not found
world: not found
```

</ChainedSnippets>
````